packages/calcite-components/src/components/stepper/stepper.tsx

```diff
@@ -338,38 +396,73 @@ export class Stepper implements LocalizedComponent, T9nComponent {
     });
   }
 
+  private determineActiveStepper(currentActivePositionChanged = false): void {
+    const totalItems = this.items.length;
+    if (!this.elWidth || !totalItems || this.layout !== "horizontal" || totalItems === 1) {
+      return;
+    }
+
+    const activePosition = this.currentActivePosition || 0;
+    const totalMinWidthOfItems = totalItems * this.getMinWidthOfStepperItem();
+    const totalRowGap =
+      (totalItems - 1) * (parseInt(window.getComputedStyle(this.containerEl).rowGap) || 0);
+
+    if (this.elWidth <= totalMinWidthOfItems + totalRowGap) {
+      if (this.singleViewMode && !currentActivePositionChanged) {
+        return;
+      }
+
+      this.singleViewMode = true;
+
+      this.items.forEach((item: HTMLCalciteStepperItemElement, index) => {
+        if (index !== activePosition) {
+          item.setAttribute("hidden", "true");
+          item.setAttribute("aria-hidden", "true");
+        } else {
+          item.removeAttribute("hidden");
```

Comment:
All HTML elements have a [`hidden`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden) prop, so you can it directly instead of setting the attribute.